ccl/changefeedccl: TestChangefeedBareJSON failed

ccl/changefeedccl.TestChangefeedBareJSON [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/20329609?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/20329609?buildTab=artifacts#/) on master @ [8e52c0cd4235fdb0907fd87361164e147ed064bc](https://github.com/cockroachdb/cockroach/commits/8e52c0cd4235fdb0907fd87361164e147ed064bc):


```
=== RUN   TestChangefeedBareJSON
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/a2df272b68db5af56229d82d0d6ebe3f/logTestChangefeedBareJSON1366847890
    test_log_scope.go:82: use -show-logs to present logs inline
    changefeed_stmt.go:448: requireNoFeedsFail: unexpected error: recv msg error: grpc: context canceled [code 1/Canceled]
    changefeed_test.go:5149: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/a2df272b68db5af56229d82d0d6ebe3f/logTestChangefeedBareJSON1366847890
--- FAIL: TestChangefeedBareJSON (20.53s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/cdc
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedBareJSON.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/ccl/changefeedccl/helpers_test.go
+++ b/pkg/ccl/changefeedccl/helpers_test.go
@@ -938,6 +938,7 @@ func requireNoFeedsFail(t *testing.T) (fn updateKnobsFn) {
 		`connection reset by peer`,
 		`knobs.RaiseRetryableError`,
 		`test error`,
+		`context canceled`,
 	}
 	shouldIgnoreErr := func(err error) bool {
 		if err == nil || errors.Is(err, context.Canceled) {

